scripts/configure.sh

```diff
@@ -305,6 +305,10 @@ if [[ $native_build == 1 && $armhf_build == 1 ]] ; then
 fi
 
 if [ $native_build == 1 ] ; then
+    echo "Generating XML files with Jinja2..."
+    python3 src/scripts/build/package.py 
```

Comment:
Here's a point where Astrobee multi-platform support makes things more complicated. The current Astrobee FSW policy is:
- Python scripts should be written so they are portable to either Python 2.7 (Ubuntu 16.04, 18.04) or Python 3.8 (Ubuntu 20.04). Of course, this means they sometimes have to prioritize maintaining Python 2.7 compatibility over using Python 3.x best practices.
- Executable scripts should start with `#!/usr/bin/env python`. Under Ubuntu, you conventionally run `python` for Python 2.x and `python3` for Python 3.x, but when we run under Ubuntu 20.04, since it doesn't have any Python 2.x interpreter, we install a symlink `python` -> `python3` to make this work in all cases.
- Note that any Python scripts used only in the ROS2 version can safely assume they are running under Ubuntu 20.04+. They don't need to maintain Python 2.x compatibility, and should explicitly call `python3`.

Reply:
Well caught! I agree, This should be executables or, if ROS2, calling python3 directly. Will change that.